python/GafferUI/SwitchUI.py

```diff
@@ -58,9 +58,23 @@
 				of 0 chooses the first input, 1 the second and so on. Values
 				larger than the number of available inputs wrap back around to
 				the beginning.
+				""",
+
+				"nodule:type", "",
+
+			],
+
+			"in" : [
+
+				"description",
 				"""
+				The array of inputs to choose from. One of these is chosen
+				by the index plug to be passed through to the output.
+				""",
+
+				"nodule:type", "GafferUI::CompoundNodule",
```

Comment:
Don't most input arrays get some extra spacing argument? The nodules on `Switch` seem really close together compared to a `SceneSwitch` or `ImageSwitch`. Actually, now that I look at it, those ones are really wide compared to `TaskSwitch` and `TaskList`...